docs/source/contributor-guide/quarterly_roadmap.md

```diff
@@ -21,6 +21,12 @@
 
 A quarterly roadmap will be published to give the DataFusion community visibility into the priorities of the projects contributors. This roadmap is not binding.
 
+## 2023 Q4
```

Comment:
I think it makes sense to keep the most recent updates at the top

Reply:
Keep the old 2022 Q2 entry?

Reply:
I don't have a strong preference one way or the other. We can either keep the old entries at the bottom of the list (as you have done in this PR) or remove them.